openshift/datasync-showcase.yml

```diff
@@ -5,10 +5,10 @@ kind: Template
 labels:
   template: ionic-showcase-server
 metadata:
-  name: mobile-developer-services-datasync-demo-server
+  name: datasync-showcase-server
   annotations:
-    openshift.io/display-name: Mobile Developer Services Data Sync Showcase
-    description: Mobile Developer Services Showcase Server connecting to Postgres Database
+    openshift.io/display-name: Data Sync Showcase 
+    description: Mobile Developer Services DataSync Showcase Server
```

Comment:
should it be `Data Sync` rather than `DataSync`, just to keep it consistent? Also I think we can drop the `Mobile Developer Services` prefix

Reply:
Done